web-components/src/components/action/action.stories.tsx

```diff
@@ -1,5 +1,4 @@
-import * as React from 'react';
-import Action from 'web-components/lib/components/action';
+import Action from '.';
```

Comment:
Wanted to do the same thing, importing components from lib in stories makes work in Storybook a bit frustrating because we have to wait for the build to complete to get changes updated.